packages/metro-transformer/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@lingui/metro-transformer",
+  "version": "0.1.0",
+  "description": "Metro bundler transformer for LinguiJS catalogs",
+  "exports": {
+    "./expo": {
+      "require": "./dist/expo/index.cjs",
+      "import": "./dist/expo/index.mjs",
+      "types": "./dist/expo/index.d.ts"
+    },
+    "./react-native": {
+      "require": "./dist/react-native/index.cjs",
+      "import": "./dist/react-native/index.mjs",
+      "types": "./dist/react-native/index.d.ts"
+    }
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "sideEffects": false,
+  "author": {
+    "name": "Vojtech Novak",
+    "email": "vonovak@gmail.com"
+  },
+  "license": "MIT",
+  "keywords": [
+    "metro",
+    "react native",
+    "metro bundler",
+    "i18n",
+    "internationalization",
+    "i10n",
+    "localization",
+    "i9n",
+    "translation",
+    "multilingual"
+  ],
+  "scripts": {
+    "build": "rimraf ./dist && unbuild",
+    "stub": "unbuild --stub"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/lingui/js-lingui.git"
+  },
+  "bugs": {
+    "url": "https://github.com/lingui/js-lingui/issues"
+  },
+  "engines": {
+    "node": ">=18.0.0"
```

Comment:
Currently, all the other packages require Node 16. I guess it would be better to do the same here if there are no blockers

Reply:
done